
int64 id
bool is_remote_controlled
float64 v
float64 w
